src/engine/bank20.asm

```diff
@@ -533,17 +684,730 @@ MapDataPointers: ; 80e5d (20:4e5d)
 	dw MapDataPointers_81697
 
 MapDataPointers_80e67: ; 80e67 (20:4e67)
-	INCROM $80e67, $8100f
+	db $1b, $59, $00, $00
+	db $22, $5a, $00, $00
+	db $13, $5c, $00, $01
+	db $2e, $5d, $00, $01
+	db $d1, $5e, $00, $01
+	db $f5, $5e, $00, $01
+	db $26, $5f, $00, $01
+	db $eb, $5f, $00, $01
+	db $43, $61, $00, $01
+	db $50, $61, $00, $01
+	db $60, $61, $00, $02
+	db $22, $62, $00, $02
+	db $36, $63, $00, $03
+	db $00, $64, $00, $03
+	db $1d, $65, $00, $03
+	db $e7, $65, $00, $03
+	db $04, $67, $00, $03
+	db $ce, $67, $00, $03
+	db $eb, $68, $00, $03
+	db $b5, $69, $00, $03
+	db $d2, $6a, $00, $03
+	db $9c, $6b, $00, $03
+	db $b9, $6c, $00, $03
+	db $83, $6d, $00, $03
+	db $a0, $6e, $00, $03
+	db $6a, $6f, $00, $03
+	db $87, $70, $00, $03
+	db $51, $71, $00, $03
+	db $6e, $72, $00, $03
+	db $21, $73, $00, $03
+	db $24, $74, $00, $04
+	db $45, $75, $00, $04
+	db $db, $76, $00, $05
+	db $8c, $77, $00, $05
+	db $8d, $78, $00, $06
+	db $d6, $79, $00, $06
+	db $00, $40, $01, $07
+	db $88, $41, $01, $07
+	db $bb, $43, $01, $08
+	db $33, $45, $01, $08
+	db $2e, $47, $01, $09
+	db $d8, $48, $01, $09
+	db $73, $4b, $01, $0a
+	db $6f, $4c, $01, $0a
+	db $fe, $4d, $01, $0b
+	db $1d, $4f, $01, $0b
+	db $b6, $50, $01, $0c
+	db $91, $51, $01, $0c
+	db $15, $53, $01, $0d
+	db $b3, $54, $01, $0d
+	db $0a, $57, $01, $0e
+	db $ce, $57, $01, $0e
+	db $f1, $7b, $00, $0e
+	db $03, $7c, $00, $0e
+	db $ef, $58, $01, $0f
+	db $79, $5a, $01, $0f
+	db $1a, $7c, $00, $0f
+	db $26, $7c, $00, $0f
+	db $e2, $5c, $01, $10
+	db $f4, $5d, $01, $10
+	db $7c, $5f, $01, $11
+	db $7f, $60, $01, $11
+	db $36, $7c, $00, $12
+	db $7d, $61, $01, $12
+	db $93, $61, $01, $12
+	db $a9, $61, $01, $12
+	db $bf, $61, $01, $12
+	db $d5, $61, $01, $12
+	db $eb, $61, $01, $12
+	db $01, $62, $01, $12
+	db $17, $62, $01, $13
+	db $da, $62, $01, $13
+	db $64, $63, $01, $13
+	db $43, $64, $01, $13
+	db $df, $64, $01, $14
+	db $b5, $65, $01, $14
+	db $47, $66, $01, $15
+	db $b8, $66, $01, $16
+	db $3e, $67, $01, $17
+	db $af, $67, $01, $18
+	db $33, $68, $01, $19
+	db $a4, $68, $01, $1a
+	db $25, $69, $01, $1b
+	db $96, $69, $01, $1c
+	db $14, $6a, $01, $1d
+	db $85, $6a, $01, $1e
+	db $28, $6b, $01, $1f
+	db $99, $6b, $01, $20
+	db $34, $6c, $01, $21
+	db $a5, $6c, $01, $22
+	db $37, $6d, $01, $23
+	db $cc, $6d, $01, $24
+	db $8a, $6e, $01, $25
+	db $18, $6f, $01, $25
+	db $c0, $6f, $01, $25
+	db $4f, $70, $01, $26
+	db $a5, $71, $01, $27
+	db $97, $73, $01, $28
+	db $b7, $73, $01, $29
+	db $e5, $73, $01, $2a
+	db $13, $74, $01, $2b
+	db $38, $75, $01, $2c
+	db $9f, $76, $01, $2d
+	db $f6, $76, $01, $2d
+	db $7c, $77, $01, $2e
+	db $c4, $77, $01, $2f
 
 MapDataPointers_8100f: ; 8100f (20:500f)
```

Comment:
Do we know what the purpose of this table is compared to `MapDataPointers_80e67`?

Reply:
Same as above except GetMapDataPointer has to be called with 2, which doesn't I didn't find yet, except for the WRAM address wd131.